requirements.txt

```diff
@@ -1 +1,2 @@
 kayobe@git+https://github.com/stackhpc/kayobe@stackhpc/yoga
+git+https://github.com/stackhpc/ansible-modules-hashivault@stackhpc
```

Comment:
We need that only for old Ansible, maybe:

ansible-modules-hashivault@git+https://github.com/stackhpc/ansible-modules-hashivault@stackhpc;python_version < "3.8"
ansible-modules-hashivault;python_version >= "3.8"

Would be better?